index.html

```diff
@@ -519,12 +519,14 @@ <h6>JWT Decoding</h6>
       </section>
     </section>
 
+    <section id="version-1.1">
+      <h2>Version 1.1</h2>
     <section id="production">
       <h2>Production</h2>
 
       <p>
         This section describes how to produce a VC-JWT encoded
-        <code>VerifiableCredential</code> from a <code>Credential</code>.
+        <code>VerifiableCredential</code> from an object of media type <code>application/credential+ld+json</code>.
```

Comment:
replace ambiguous "Credential" with specific media type, this change is potentially blocked by https://github.com/w3c/vc-data-model/pull/1014

If that PR changes substantially, this PR will need to be rejected / reworked.